cli/cmd/upload.go

```diff
@@ -176,20 +57,22 @@ var uploadCmd = &cobra.Command{
 			nexusUrl = url
 		}
 
-		req, err := http.NewRequest("POST", nexusUrl, requestBody)
+		nexusUrl = fmt.Sprintf("%s/nais/%s/%s/nais.yaml", strings.TrimRight(nexusUrl, "/"),
```

Comment:
I would prefer having `strings.TrimRight(nexusUrl, "/")` on a separate line, as the `fmt.Sprintf` is pretty complex from before.